content/docs/projects/csi-driver.md

```diff
@@ -80,17 +68,29 @@ items:
 ...
 ```
 
-The CSI driver is now installed and is ready to be used for pods in the cluster.
+### Kubernetes Requirements
```

Comment:
I think this section is not very useful anymore, no-one should be running cert-manager 1.15/1.16 anymore (no public cloud provider still supports these versions).